### ISSUE TITLE
Rasters not curated in ArcRave for VBET

### ISSUE THREAD

**Issue by lauren-herbine** [Riverscapes/riverscapes-tools#555]
**I am using Rave version 3.0.3.0 and ArcMap 10.8.1
I am using [this project](https://data.riverscapes.xyz/#/NRCS_CEAP/4811ec77-78e0-4449-a36b-b03428b7be85).**
@shelbysawyer see this video for the issue:
https://youtu.be/vhhBUL8-2-w

When I open it using ArcRave, the likelihood of being a valley bottom raster is symbolized on a grayscale, and the geomorphic units come in with default symbology:
![image](https://user-images.githubusercontent.com/73319684/155175885-8edf6568-ae1c-43a5-b308-8673483845b0.png)

Arc is also throwing a lot of drawing errors, which maybe has something more to do with Rave? I explain some of that in the video. I'll tag @MattReimer on this as well.
![image](https://user-images.githubusercontent.com/73319684/155176484-d35425e9-58d3-42f8-a00b-fb6577eb8099.png)

@joewheaton have you opened VBET in Arc recently? Any ideas?

**Comment by lauren-herbine** [comment]
Just watched the video and it looks like there is no sound :( Let me know if the messages aren't coming across and I can re-do

**Comment by shelbysawyer** [comment]
@lauren-herbine I have not encountered this and I've been in ArcRAVE VBET quite a bit over the past few weeks. Let me explore it a bit and get back to you! Thanks Lauren!

**Comment by shelbysawyer** [comment]
I've made a quick 2 min video with my thoughts on what's happening:

[![Not showing https://youtu.be/UzCmRVYofMQ](http://img.youtube.com/vi/UzCmRVYofMQ/0.jpg)](http://www.youtube.com/watch?v=UzCmRVYofMQ"")

**These are the key points:**
1. I've been working in RAVE 3.1.0 because it's the most recent update that I saw, but now I [see](https://github.com/Riverscapes/RaveAddIn/releases) that it's been labeled as a "Pre-Release" and that RAVE 3.0.3 is labeled as "Latest". @philipbaileynar @joewheaton Should I be working in 3.0.3 instead?
2. This project is relatively large (4.4 GB). I've noticed that some larger projects get interrupted at some point in the download or unzipping process and this can lead the projects not to work properly. If the issue is not your version of RAVE, I would try re-downloading and unzipping it again.
3. The drawing issue is a known problem (I can't find the darn issue at the moment, but I'll look around some more).

Hope this helps!

**Comment by lauren-herbine** [comment]
@shelbysawyer Thanks! It looks like I've been working in "latest" instead of "pre-release", and that is where the bulk of my issues lie. Like you said, the drawing issue is still a thing, but this solves most of my problems.

Thanks! Closing

**Comment by shelbysawyer** [comment]
@lauren-herbine raised an important point when we chatted about this: the RAVE website [page](https://rave.riverscapes.xyz/Download/install_arcrave.html) for installing / upgrading ArcRAVE has the 3.0.3 version of RAVE provided in the "download" [link](https://github.com/Riverscapes/RaveAddIn/releases/tag/3.0.3). The other releases aren't visible at this link. Maybe we can update this link to drop users at the release page (not a version-specific one)?

[![Not showing https://youtu.be/0eda7lLNDAI](http://img.youtube.com/vi/0eda7lLNDAI/0.jpg)](http://www.youtube.com/watch?v=0eda7lLNDAI"")

**Comment by joewheaton** [comment]
@lauren-herbine did you by chance forget to hit "Update Resources"?  Great suggestions @shelbysawyer on your video (i.e. update to latest"). 

@MattReimer is there a reason that 3.1.0 is a Pre-Release? I think that we still want the link to go to "Latest" in @shelbysawyer last video, but why are we not promoting 3.1?

**Comment by lauren-herbine** [comment]
@joewheaton no, I tried updating my resources before downloading the pre-release. It didn't make any difference- the only thing that worked was switching from "latest" to "pre-release"!
